content/en/getting_started/tracing/_index.md

```diff
@@ -20,159 +20,203 @@ further_reading:
 
 ## Overview
 
-Datadog Application Performance Monitoring (APM or tracing) is used to collect [traces][1] from your backend application code. This beginners' guide shows you how to get your first trace into Datadog.
+Datadog Application Performance Monitoring (APM) provides deep visibility into your applications, enabling you to identify performance bottlenecks, troubleshoot issues, and optimize your services.
 
-**Note**: Datadog APM is available for many languages and frameworks. See the documentation on [Instrumenting Your Application][2]
+This guide demonstrates how to get started with APM and send your first trace to Datadog.
 
-## Datadog account
+At a high level, you need to:
 
-If you haven't already, create a [Datadog account][3].
+1. Set up Datadog APM to send observability data to Datadog.
+1. Run your application to generate data.
+1. Explore the collected data in Datadog.
 
-## Datadog Agent
+## Prerequisites
 
-Before installing the Datadog Agent, set up a [Vagrant Ubuntu 22.04 virtual machine][4] using the following commands. For more information about Vagrant, see their [Getting Started][5] page.
+To complete this guide, you need the following:
 
-```text
-vagrant init ubuntu/jammy64
-vagrant up
-vagrant ssh
-```
+1. [Create a Datadog account][1] if you haven't already.
+1. Find or create a [Datadog API key][2].
+1. Start up a Linux host or VM.
 
-To install the Datadog Agent on a host, use the [one line install command][6] updated with your [Datadog API key][7]:
+## Set up Datadog APM
 
-```shell
-DD_API_KEY=<DATADOG_API_KEY> DD_SITE="{{< region-param key="dd_site" >}}" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
-```
+To set up Datadog APM without needing to touch your application code, use Single Step APM Instrumentation:
```

Comment:
Maybe "without needing to modify your application's code or the deployment process"? Want to emphasize that nothing is changed other than installing the datadog agent.